src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  left: 0;
+  top: 0;
+  border-radius: 4px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```

Comment:
you should set box-sizing with `*` selector, since it should be the same for all elements
```suggestion
```